src/com/lilithsthrone/game/character/body/types/BodyPartTypeInterface.java

```diff
@@ -131,4 +132,15 @@ public default String getTransformName() {
 	public default BodyPartClothingBlock getBodyPartClothingBlock() {
 		return null;
 	}
+
+	/**
+	 * Checks, if the given BodyPartType is in the list of BodyPartTypes
+	 * @param values The list of BodyPartTypes to match the type against
+	 * @return true, if the type is among the list.
+	 */
+	public default boolean isOneOf(BodyPartTypeInterface... values) {
+		ArrayList<? extends BodyPartTypeInterface> list = new ArrayList<>(Arrays.asList(values));
+		list.removeIf(e -> e==null);
```

Comment:
I don't think removing nulls is necessary, as `contains()` shouldn't be tripped up by it, and this results in walking the list an extra time (even if it is always a very short list). If this is indeed the case, I think the entire function would decompose to just the one-liner `Arrays.asList(values).contains(this)`. (The creation of a new ArrayList is unnecessary here, since the list is not modified.)

If I am wrong, then the first line could use `Util.newArrayListOfValues`, but that change would be of lesser importance.

Reply:
Well, The case `Arrays.asList(null).contains(null)` would theorhetically return true but that can't happen, since `this` can't be `null` without causing an NPE.